pkg/logs/tailers/socket/tailer.go

```diff
@@ -67,7 +68,10 @@ func (t *Tailer) forwardMessages() {
 	}()
 	for output := range t.decoder.OutputChan {
 		if len(output.GetContent()) > 0 {
-			t.outputChan <- message.NewMessageWithSource(output.GetContent(), message.StatusInfo, t.source, output.IngestionTimestamp)
+			origin := message.NewOrigin(t.source)
+			source_host_tag := fmt.Sprintf("source_host:%s", t.Conn.RemoteAddr().String())
```

Comment:
Since this isn't a hostname, and also includes the port number, would it make sense to call this `source_address`?

Reply:
Hey Vikentiy, Carlos says that we should also be removing the port number.

Also here is the follow up discussion with him.

<img width="493" alt="image" src="https://github.com/user-attachments/assets/75cc131d-52ba-4446-8d8f-57bf7a88e84a">